srtcore/api.cpp

```diff
@@ -233,15 +228,24 @@ string srt::CUDTUnited::CONID(SRTSOCKET sock)
     return os.str();
 }
 
-int srt::CUDTUnited::startup()
+void srt::CUDTUnited::implicitStartup()
```

Comment:
Once you have a parameter in `startup()`, this function is no longer necessary. You can call it as `startup(true)` in socket and group creation functions.